### ISSUE TITLE
android: url source causes AUDIO_OUTPUT_FLAG_FAST denied

### ISSUE THREAD

**Issue by emekalites** [bluefireteam/audioplayers#146]
I am getting this error on my android whenever i try to play a remote file url:

`AUDIO_OUTPUT_FLAG_FAST denied, rates do not match 44100 Hz, require 48000 Hz`

Android version: 8.1
I'll like some help on how to fix this, Thanks.

**Comment by erickzanardo** [comment]
This hardly seems a problem with audioplayers, most likely the problem is with file, have you tried opening that file on a media player, like VLC for example?

**Comment by emekalites** [comment]
It actually plays on ios without any errors or exceptions.

**Comment by erickzanardo** [comment]
The two platforms are not equivalent. We do not use explicitly this flag anywhere in the code, and searching for it, some Issues suggests that the device cannot play the file with any apps. So please confirm if the file is playable in the device using a well know player, like VLC, that way we can be sure if the problem is indeed with audioplayers.

Another question, are you using PlayerMode.LOW_LATENCY? This mode has a lot of limitations regarding file size, and rate.

**Comment by emekalites** [comment]
The songs are playing on vlc and i did not set player mode to PlayerMode.LOW_LATENCY in my app (I am aware of its limitations).

**Comment by erickzanardo** [comment]
Right, are you able to provide the file or its remote url so we can try to track down this issue? I still thinks there is something unique to this file that makes audiplayers break because other files works fine.

**Comment by emekalites** [comment]
Hi @erickzanardo 
This is a sample url: **https://aws.shuzia.com/file/2019_04_16_Biigmanie-M1555371317.mp3?dir=1555881297&uu=3154**

I have tried different scenarios. 
1) get the app to play the audio file remotely using `final result = await _audioPlayer.play('https://aws.shuzia.com/file/2019_04_16_Biigmanie-M1555371317.mp3?dir=1555881297&uu=3154', isLocal: false);` -- did not work
2) get the app to download the file and play it using `final result = await _audioPlayer.play('urlFromLocalPath', isLocal: true);` --- did not work
3) I downloaded the file myself and put it in the assets directory, then used `audioCache.play('2019_04_16_Biigmanie-M1555371317.mp3')` --- it worked

Well, only scenario number 3 worked which is not exactly how the app is designed to function. My app needs to play the audio file either through scenario number 1 or 2.
